sklearn/datasets/descr/diabetes.rst

```diff
@@ -2,7 +2,37 @@ Diabetes dataset
 
 Notes
 -----
+
 Ten baseline variables, age, sex, body mass index, average blood
 pressure, and six blood serum measurements were obtained for each of n =
 442 diabetes patients, as well as the response of interest, a
 quantitative measure of disease progression one year after baseline.
+
+Note that the 10 x variables have been standardized to have mean 0 and squared length = 1 (sum(x^2)=1).
```

Comment:
Why the "x" in "10 x variables"? I would rather just say "10 input variables" if the goal is to tell them apart from the target variable.

Also `mean 0 and squared length = 1 (sum(x^2)=1)` is not clear to me. Is this per-variable standardization (per variable mean removal and unit variance scaling)? Or is it row-based normalization? The later would be counter-intuitive to me but this is what the use of the word "length" would suggest to me.